External/Plugins/FlashDebugger/Debugger/WatchManager.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using PluginCore;
+using PluginCore.Helpers;
+using PluginCore.Utilities;
+
+namespace FlashDebugger.Debugger
+{
+    public class WatchManager
+    {
+        private IProject m_Project;
+        private string m_SaveFileFullPath;
```

Comment:
Is `m_` the coding convention for this plugin? That's certainly not what we usually follow.